src/ngx_http_lua_util.c

```diff
@@ -3813,6 +3820,7 @@ ngx_http_lua_init_vm(lua_State *parent_vm, ngx_cycle_t *cycle,
     ngx_pool_t *pool, ngx_http_lua_main_conf_t *lmcf, ngx_log_t *log,
     ngx_pool_cleanup_t **pcln)
 {
+    int                             rc;
```

Comment:
Alignment issue. Better use `ngx-releng` to check coding style, always.

Reply:
Fixed - sometimes I forget :)